libs/langchain/langchain/document_loaders/airbyte_cdk.py

```diff
@@ -0,0 +1,27 @@
+"""Loads local airbyte json files."""
+import json
+from typing import Any, Iterator, List, Mapping, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+from airbyte_protocol.models.airbyte_protocol import AirbyteRecordMessage, AirbyteStateMessage
+from airbyte_cdk.sources.embedded.base_integration import BaseEmbeddedIntegration
+from airbyte_cdk.sources.embedded.runner import CDKRunner
+
+
+class AirbyteCDKLoader(BaseLoader, BaseEmbeddedIntegration):
+    """Loads records using an Airbyte source connector implemented using the CDK."""
+
+    def __init__(self, config: Mapping[str, Any], source_class: Any, stream_name: str, state: Optional[AirbyteStateMessage] = None) -> None:
+        super().__init__(config=config, runner=CDKRunner(source=source_class(), name=source_class.__name__))
+        self._stream_name = stream_name
+        self._state = state
```

Comment:
This might be something on the CDK runner side, but do we want to do any sort of validation on the config? I think if we're not doing so already, a quick win on UX could be to run `check` with the config on init